be/src/column/column.h

```diff
@@ -152,6 +152,17 @@ class Column {
 
     virtual void append(const Column& src) { append(src, 0, src.size()); }
 
+    // TODO(fzh): optimize replicate() for specific columns, such as array_column, object_column.
+    virtual ColumnPtr replicate(const std::vector<uint32_t>& offsets) {
```

Comment:
should add some comments to explain what this function is for.

Reply:
added, to aligned with array's offset in lambda functions.